docs/src/design/chiplets/main.md

```diff
@@ -160,6 +161,8 @@ The bus is implemented as a single [running product column](../lookups/multiset.
 - Each chiplet response is “sent” by computing the same operation-specific lookup value from the label, inputs, and outputs, and then multiplying it into the $b_{chip}$ running product column.
 
 Thus, if the requests and responses match, then the bus column $b_{chip}$ will start and end with the value $1$. This condition is enforced by boundary constraints on the $b_{chip}$ column.
+It is also possible to invoke chiplet computations through public inputs, by initializing the bus with requests that must be responded to by the chiplets.
+In this case, the verifier derives the initial value of the bus as the inverse product of all reduced requests.
```

Comment:
nit: I think we can either initialize the requests or responses and hence using the inverse might not be required, or is there something else?

Reply:
Indeed, that makes sense, chiplets should only use responses and this forces the initialization with requests.
Could we add a note clarifying this?